src/redux/user/actions.js

```diff
@@ -36,3 +52,35 @@ export const requestUser = id => dispatch => {
     })
     .catch(err => dispatch(requestUserFail(err)));
 };
+
+export const requestUserUpdate = data => dispatch => {
+  dispatch(requestUserStart());
+  fetch(`${process.env.REACT_APP_API_BASE_URL}/api/user`, {
+    method: "PUT",
+    credentials: "include",
+    headers: {
+      "Content-Type": "application/json"
+    },
+    body: JSON.stringify(data)
+  })
+    .then(handleResponse)
+    .then(data => dispatch(requestUserSuccess(data)))
+    .catch(err => dispatch(requestUserFail(err)));
+};
+
+export const requestAvatarUpdate = data => dispatch => {
+  dispatch(requestAvatarUpdateStart());
+  const test = new FormData();
+  test.append("file", data);
+

```

Comment:
Whoops, looks like you forgot to change the `test`-variables to something more declarative :)

Reply:
Agree.